### PR TITLE
Educacao

### PR DESCRIPTION
**Pull request by dbattistelo**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: portugues_pt/educacao/educacao.md [hunk 1]
```diff
@@ -0,0 +1 @@
+UM povo somente evolui com Educação!!!
```
Comment (casey-collab) [inline portugues_pt/educacao/educacao.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**